example/src/shared/Wysiwyg/MediaLibrary.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { ModalLayout, ModalHeader, ModalBody, ModalFooter } from '@strapi/parts/ModalLayout';
+import { Text, Button } from "@strapi/parts";
+
+const image = [{alt: 'sunrise', url: 'http://localhost:3000/sunriseimage'}, {alt: 'sunset', url: 'http://localhost:3000/sunsetimage'}]
+
+const MediaLibrary = ({ setMediaLibVisible, handleSubmitImage }) => {
```

Comment:
`const MediaLibary = ({ onToggle, onSubmitImage }) => {}`

Can you also add propTypes?

Reply:
It is great to follow [this convention](https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364) when naming props.